Sources/MQ/Utilities/Identity.swift

```diff
@@ -0,0 +1,13 @@
+/// Convenience helper for identity function.
+///
+/// Identity function is a function which returns
+/// exactly the same value as provided as argument
+/// without any modifications.
+///
+/// - Parameter value: Value forwarded.
+/// - Returns: Value provided as an argument without any changes.
+public func id<Value>(
```

Comment:
So you just basically wrote the method for returning the parameter passed as it's argument.

I think i don't quite get it's usefulness and potential use cases. Can you elaborate a little bit on that?

Reply:
it is useful when you need a function returning the same value i.e. 
```swift
map { $0 }
```
will be instead:
```swift
map(id)
```